serenity-reports/src/main/java/net/thucydides/core/reports/html/Reporter.java

```diff
@@ -39,11 +39,13 @@ private void generateReports() {
             final List<ReportExecutor> partitions
                     = reportingTasks.stream()
                     .map(ReportExecutor::new)
+                    .filter(data-> data != null)
```

Comment:
Is It possible that anytime the reportingTasks assigned in Line 25 have null values??
@wakaleo